azure-pipelines-ui-tests.dfe.yml

```diff
@@ -0,0 +1,212 @@
+
+schedules:
+- cron: 0 5 * * 1,2,3,4,5
+  branches:
+    include:
+    - dev
+  always: true
+- cron: 0 19 * * 1,2,3,4,5
+  branches:
+    include:
+    - refs/heads/dev
+  always: true
+name: $(date:yyyyMMdd)$(rev:.r)
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: refs/heads/dev
+jobs:
+- job: Job_2
+  displayName: Public UI tests
+  timeoutInMinutes: 160
+  cancelTimeoutInMinutes: 10
+  condition: succeededOrFailed()
+  pool:
+    vmImage: ubuntu-20.04
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 5
+    fetchTags: false
+  - task: UsePythonVersion@0
+    displayName: Use Python 3.10
+    retryCountOnTaskFailure: 5
+    timeoutInMinutes: 5
+    inputs:
+      versionSpec: 3.10
+  - task: AzureKeyVault@2
```

Comment:
Would suggest adding spaces between `-task` & `-job` blocks to make them more readable

Reply:
This hasn't been resolved